core/src/test/java/nl/nn/adapterframework/configuration/ClassLoaderManagerTest.java

```diff
@@ -94,16 +94,16 @@ else if(type.isEmpty()) //If empty string, it's a WebAppClassLoader
 
 		if(type.endsWith("DirectoryClassLoader")) {
 			String directory = getTestClassesLocation()+"ClassLoader/DirectoryClassLoaderRoot/";
-			appConstants.put("configurations."+configurationName+".directory", directory);
+			appConstants.put("configurations."+configurationName+".directory", (Object) directory);
```

Comment:
Waarom is de cast naar Object nodig?

Reply:
Ja ik wist niet goed hoe ik dit op zou moeten pakken, je wilt de property alleen in de local appconstants toevoegen. Ik wou hier eerst een put methode voor aanmaken maar toen bedacht ik mij dat dat alleen maar problemen kan veroorzaken en het zonder gemakkelijker/leesbaarder en meer dummy-safe zou zijn. Deze put(object, object) is deprecated, dus hij wordt (hopelijk) door niemand anders gebruikt!